src/filebrowser/browser.ts

```diff
@@ -112,10 +112,11 @@ class FileBrowserWidget extends Widget {
     this._listing = new DirListing({ manager, model, opener, renderer });
 
     model.fileChanged.connect((fbModel, args) => {
+      let oldPath = args.oldValue ? args.oldValue.path : void 0;
```

Comment:
Since I broke this I would like to understand the logic. Are you making sure the oldValue exists and has the path?